app/src/androidTest/java/org/wikipedia/TestLogRule.kt

```diff
@@ -0,0 +1,35 @@
+package org.wikipedia
+
+import android.util.Log
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class TestLogRule : TestRule {
+    override fun apply(base: Statement?, description: Description?): Statement {
+        return object : Statement() {
+            override fun evaluate() {
+                try {
+                    base?.evaluate()
+                } catch (t: Throwable) {
+                    Log.e("orange", "test rule")
```

Comment:
What's "orange"? Do we need this log line at all?

Reply:
when i was testing the log i usually label it as orange, so that i can filter easily in log. Forgot to remove this. Its removed.